Update Death Star冥星.md

HanSaMoLi opened the pull request.

HanSaMoLi committed:
Update Death Star冥星.md

BRC1024Rootverse commented:
這個沒有看到有提交的修改哦！

在這一篇上，可以來做一些交流哦～

您好，
我看您幾乎評論了所有的星球，給每個星球寫了一個概念，所以想詢問您對這個地圖的一些看法。

期待您的回覆。

HanSaMoLi commented:
不好意思，今天有些忙。刚回家，您的邮件我都看了，地图的问题，我的想法是这个宇宙诞生，各个星域的各自特征，导致了不同种族的诞生，并且不同种族之间的故事、历史交织在一起。
        麻烦问一下，您有其他的交流方式吗？或者还是用邮件方便一些？

BRC1024Rootverse ***@***.***>于2024年7月22日 周一13:23写道：

> 這個沒有看到有提交的修改哦！
>
> 在這一篇上，可以來做一些交流哦～
>
> 您好，
> 我看您幾乎評論了所有的星球，給每個星球寫了一個概念，所以想詢問您對這個地圖的一些看法。
>
> 期待您的回覆。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BRC1024Rootverse/Rootverse-Map/pull/8#issuecomment-2242113861>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BHLCXFKJP3ZYHNS2GILJUDDZNSJNTAVCNFSM6AAAAABLBJB776VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDENBSGEYTGOBWGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

HanSaMoLi committed:
Update Death Star冥星.md

Death Star is a hauntingly mysterious planet shrouded in perpetual darkness and eerie silence. It is home to ancient tombs, spectral swamps, and desolate plains, where the undead roam freely and the air is thick with the whispers of lost souls. This enigmatic world is ruled by powerful necromantic forces, embodying the very essence of death and the afterlife.
冥星是一个充满神秘和恐怖的星球，常年笼罩在永恒的黑暗和诡异的寂静中。这里有古老的墓地、幽魂沼泽和荒凉的平原，亡灵自由游荡，空气中弥漫着失落灵魂的低语。这个神秘的世界由强大的死灵力量统治，象征着死亡和来世的本质。

HanSaMoLi commented:
您好，已经全部修改完成，请审阅。

BRC1024Rootverse commented:
您好呀～

 沒關係的，想問問您有沒有興趣一起建設Rootverse的地圖，也可以把您的想法慢慢的在地圖裡完善。
我已經問了空空您的聯繫方式，我們稍微微信或者TG聊！
